### PR TITLE
vae config and dataset files

### PR DESCRIPTION
**Pull request by Shunili**

### CODE REVIEW

**Comment by sparticlesteve** [comment]
looking pretty reasonable to me. Have you done any testing yet?
I think you need a get_model function in models/vae.py.

File: datasets/__init__.py [hunk 1]
```diff
@@ -22,11 +22,14 @@ def get_data_loaders(name, batch_size, cache_path = None, distributed=False,
     """Construct training and validation datasets and data loaders"""
 
     # Get the datasets
-    
-    #add: check if cached
     print('load datasets')
-    if cache_path is None:
+    if cache_path==None:
```
Comment (sparticlesteve) [inline datasets/__init__.py:26]: minor thing unrelated to your PR but in python it's recommended to use "is" here for checks again None: https://www.python.org/dev/peps/pep-0008/#programming-recommendations

Comment (Shunili) [reply datasets/__init__.py:26]: Ah I see. There are a few other places that uses ==. I'll go fix them.

**Comment by sparticlesteve** [comment]
should I merge now? I might have time to test things this week.

**Comment by Shunili** [comment]
Yes, you could merge now. Just updated a  few things in the notebook.

**Pull request merged**